tests/scalers/aws/aws_managed_prometheus/aws_managed_prometheus_test.go

```diff
@@ -0,0 +1,190 @@
+//go:build e2e
```

Comment:
Does it make sense adding a test case using podIdentity?

Reply:
added tests/scalers/aws/aws_managed_prometheus_pod_identity/aws_managed_prometheus_pod_identity_test.go